src/test/resources/com/puppycrawl/tools/checkstyle/checks/design/finalclass/InputFinalClassPrivateCtor.java

```diff
@@ -0,0 +1,101 @@
+/*
+FinalClass
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.design.finalclass;
+
+public class InputFinalClassPrivateCtor {
+
+    private  class some1 { // violation 'Class some1 should be declared as final'
```

Comment:
`Some1` keep naming  in correct case(CamelCase) to ease read code.

apply for all others you introduce.

Reply:
done